src/rise-twitter.html

```diff
@@ -1,8 +1,2 @@
-<style>
-  :host {
-    display: flex;
-  }
-</style>
-<div id="twitter-component-template">
-  <h1>Rise Twitter Component</h1>
-</div>
+<link rel="stylesheet" type="text/css" href="../dist/static/css/main.css">
```

Comment:
@jungeunyoon I am not sure this href path will work on the deployed component in player. Shouldn't it be changed by webpack depending if it is a dev or prod build? cheers